app/react-native/readme.md

```diff
@@ -90,7 +82,7 @@ The following parameters can be passed to the start command:
     host to listen on
 -p, --port <port>
     port to listen on
--s, --secured
+--https
```

Comment:
Needs a note in `MIGRATION.md` and/or should be deprecated with one major version of back-compat

Reply:
I will add it to the MIGRATION.MD